### DISCUSSION TITLE
Samba Share Support

### DISCUSSION THREAD

**Discussion by mathieugoodman** [vicwomg/pikaraoke#200] [Ideas]
Does pikaraoke currently support Samba Shares? If not, is it something that could eventually be supported? I would love to have my karaoke collection on a USB drive connected to my router that supports Samba, and access it from any device on my network that supports pikaraoke. Thanks!

**Comment by vicwomg** [comment]
You can configure pikaraoke to point to any source/download directory using the -d option. So technically it should "just work", assuming you've set up the mounting and permissions correctly. There should be some good tutorials on how to get a samba mount configured on boot, and then you'd run pikaraoke with `sudo python3 app.y -d <samba_path>`

**Reply by mathieugoodman** [reply]
Thank you! I ended up just plugging my USB hard drive directly into my RPi for now and everything is working nicely.

Thanks for creating such a great application. I was trying to get something similar going on Windows with Winamp, but your solution is way simpler and more elegant.
